spark_pipeline_framework/transformers/fhir_sender/v1/fhir_sender.py

```diff
@@ -437,6 +446,17 @@ def _transform(self, df: DataFrame) -> DataFrame:
                         ),
                     ).toDF(json_schema)
 
+                if debug:
+                    response_df = json_df.cache()
+                    response_df = response_df.coalesce(1)
+                    response_df.createOrReplaceTempView("result_view")
```

Comment:
This is not "response" df. Also can we use "intermediate_view" instead of "result_view"